### PR TITLE
Updated Spanish translation

### PR DESCRIPTION
**Pull request by macagua**
Updated Spanish translation for Volto

### CODE REVIEW

File: locales/es/LC_MESSAGES/volto.po [hunk 1]
```diff
@@ -4348,7 +4348,7 @@ msgstr "Comparar con el idioma"
 #: components/manage/Controlpanels/Relations/BrokenRelations
 # defaultMessage: {countofrelation} broken {countofrelation, plural, one {relation} other {relations}} of type {typeofrelation}
 msgid "countBrokenRelations"
-msgstr ""
+msgstr "{countofrelation} roto {countofrelation, plural, one {relación} other {relaciones}} de tipo {typeofrelation}"
```
Comment (erral) [inline locales/es/LC_MESSAGES/volto.po:4351]: las variables deben quedar igual relación y relaciones deben sustituirse por sus valores originales

File: locales/es/LC_MESSAGES/volto.po [hunk 2]
```diff
@@ -4443,7 +4443,7 @@ msgstr "vaciar intIds y reconstruir relaciones"
 #: helpers/MessageLabels/MessageLabels
 # defaultMessage: <ul><li>Regenerate intIds (tokens of relations in relation catalog)</li><li>Rebuild relations</li></ul><p>Check the log for details!</p><p><b>Warning</b>: If you have add-ons relying on intIds, you should not flush them.</p>
 msgid "flushAndRebuildRelationsHints"
-msgstr ""
+msgstr "<ul><li>Regenerar intIds (tokens de relaciones en el catálogo de relaciones)</li><li>Reconstruir relaciones</li></ul><p>¡Consulte el registro para obtener más detalles!</p><p><b>Advertencia</b>: Si tiene complementos que dependen de intIds, no debe eliminarlos.</p>"
```
Comment (erral) [inline locales/es/LC_MESSAGES/volto.po:4446]: debe -> debería ?

**Review by erral — CHANGES_REQUESTED** [review]

File: locales/es/LC_MESSAGES/volto.po (hunk 2 above)
Comment (macagua) [reply locales/es/LC_MESSAGES/volto.po]: Hecho con commit [ba35493]

File: locales/es/LC_MESSAGES/volto.po (hunk 1 above)
Comment (macagua) [reply locales/es/LC_MESSAGES/volto.po]: Hecho con commit [ba35493]

**Comment by erral** [comment]
Why do we have so many changes here? It looks like the PR is not properly rebased, right?

**Comment by macagua** [comment]
@erral Only I made a ``git pull origin master`` and ``git rebase master`` at ``spanish-translation`` branch

**Comment by davisagli** [comment]
@macagua @erral I rebased the PR to only include the relevant changes. I'm not sure what went wrong with your rebase before, @macagua

**Comment by davisagli** [comment]
@macagua Needs a changelog entry in the news folder

**Comment by macagua** [comment]
> @macagua Needs a changelog entry in the news folder

@davisagli done with [feeeea3](https://github.com/plone/volto/pull/5120/commits/feeeea3bd2ff9e182e97349d05b2f58d996ee4d1)

File: locales/es/LC_MESSAGES/volto.po [hunk 3]
```diff
@@ -1559,7 +1559,7 @@ msgstr "Bloque embebido de Google Maps"
 #: components/manage/Blocks/Grid/schema
 # defaultMessage: Grid
 msgid "Grid"
-msgstr ""
+msgstr "Cuadrícula"
```
Comment (sneridagh) [inline locales/es/LC_MESSAGES/volto.po:1562]: @macagua Sinceramente, yo no traduciría "Grid" y lo dejaría en inglés. El término ya es suficientemente abierto a interpretaciones y creo que intentar traducirlo induce a confusiones.

@erral Qué te parece?

**Pull request merged**